src/student_success_tool/analysis/pdp/features/student_term.py

```diff
@@ -90,6 +91,8 @@ def aggregate_from_course_level_features(
             "grade",  # TODO: only if this is actually categorical
         ],
     )
+    df_dummies[constants.NUM_COURSE_FEATURE_COL_PREFIX + '_grade_DFW'] = sum_num_dfw_courses(df_dummies)
```

Comment:
I see that this doesn't follow any existing patterns for feature calculations. That isn't necessarily a bad thing, but I am wondering if there's a way to fit this nicely into existing patterns.

Reply:
In fact... is it possible to just add this to the `agg_col_vals` list directly below, like so:

```
 agg_col_vals = [
    ...,
    ("grade", ["D", "F", "W"]),
]
```

Reply:
oh, awesome!!